src/pages/offers/offer-screen.tsx

```diff
@@ -11,18 +11,24 @@ import OfferOption from '../../components/offers/offer-option';
 import { Link, useParams } from 'react-router-dom';
 import HotelCard from '../../components/hotel-card/hotel-card';
 import { useState } from 'react';
-import {useAppSelector} from '../../hooks';
-import { changeOffers } from '../../store/reduser';
+import {useAppDispatch, useAppSelector} from '../../hooks';
```

Comment:
И везде по коду надо унифицировать, много где встречается этот косяк

```suggestion
import { useAppDispatch, useAppSelector } from '../../hooks';
```